ts/components/conversation/Message.tsx

```diff
@@ -848,6 +872,16 @@ class MessageInner extends React.PureComponent<MessageRegularProps, State> {
     );
   }
 
+  /**
+   * Doubles / halves the playback speed based on the current playback speed.
+   */
+  private updatePlaybackSpeed() {
+    this.setState({
+      ...this.state,
+      playbackSpeed: this.state.playbackSpeed === 1 ? 2 : 1,
```

Comment:
when you rely on the state of before in the setState lin a class component ike you do here with your `this.state.playbackSpeed` you need to use the other syntax of setState as

```ts
this.setState(state => {
      playbackSpeed: state.playbackSpeed === 1 ? 2 : 1,
})
```
this is to ensure the state you use is the latest


also there is no need to use the spread operator ... when dealing with those kind of states: https://stackoverflow.com/a/55247471/1680951